playwright.config.ts

```diff
@@ -1,80 +1,97 @@
-import { defineConfig, devices } from '@playwright/test';
-import 'dotenv/config';
+import { defineConfig, devices } from "@playwright/test";
+import "dotenv/config";
```

Comment:
What changed in this file? I guess you ran some lint on it? Are there any changes required for `Test: Add support for multiple users`?

Reply:
Just lint. Along with everything else that was updated with non-vs-code browsers.